pkg/webmiddleware/log.go

```diff
@@ -24,7 +25,14 @@ import (
 
 // Log returns a middleware that logs requests.
 // If logger is nil, the logger will be extracted from the context.
-func Log(logger log.Interface) MiddlewareFunc {
+func Log(logger log.Interface, ignorePaths []string) MiddlewareFunc {
+	ignoreRegex := []*regexp.Regexp{}
+	for _, path := range ignorePaths {
+		ignoreRegex = append(ignoreRegex, regexp.MustCompile(path))
```

Comment:
Don't panic here; I think that's going to be really hard for users to figure out why there was a panic in the middle of all the configuration setup for TTS. This means that `Log()` should return the error or that it warns here that a regex is invalid and execution continues